### ISSUE TITLE
Performance not improved with several training configs and modifications

### ISSUE THREAD

**Issue by chau25102001** [hukaixuan19970627/yolov5_obb#654]
Hi, I want to thank you for your great work.
Recently, I've used YOLOv5-obb for training on my custom dataset with 2 bounding box classes. The issue is the inference's False Negative (FN) is too high, and I cannot improve the performance after many trials, I've tried to change some configurations, such as: learning rate, weight decay, even adding my own AdamW optimizer. My data is a combination of a private set and a public set called Retail50K (example below) and I cannot share here, but it is for groceries rows detection, and the objects (rows) bounding box labels often horizontal and long (many x coordinates is equal to 0 or image width), so I also remove the label mask in your dataset code to make sure no labels are removed:
```
            labels_mask = (rboxes[:, 0] >= 0) & (rboxes[:, 0] < img.shape[1]) \
                        & (rboxes[:, 1] >= 0) & (rboxes[:, 0] < img.shape[0]) \
                        & (rboxes[:, 2] > 5) | (rboxes[:, 3] > 5)
            labels_obb = labels_obb[labels_mask]
```
I also used multi-scale training as well as your provided pretrained weights for yolov5m but the problem does not seemed to be solved. What could be the issue here? Here is an example of Retail50K dataset:
![image](https://github.com/hukaixuan19970627/yolov5_obb/assets/79529778/5a2203f9-0cf4-4222-b69d-d717a7605c75)
I used the image size of (640, 640) for training.
